django_urlqueryset/models.py

```diff
@@ -0,0 +1,52 @@
+import json
+from urllib import parse
+
+import requests
+from django.core.serializers import json as json_serializer
+from django.db import models
+
+from django_urlqueryset.utils import get_default_params
+
+
+class UrlModel(models.Model):
+    class Meta:
+        abstract = True
+
+    def get_params(self):
+        params = get_default_params()
+        params.pop('fetch_method')
```

Comment:
Why you need to know there is a key called fetch_method in params and pop it?
If u need this pop please add a default return value for the pop to forget KeyError Exception.